### PR TITLE
feat: add cloudRegistration and Registration servicers to bootstrapper service

### PR DESCRIPTION
**Pull request by reginawang3495**
Signed-off-by: Regina Wang <rwang3495@gmail.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go [hunk 1]
```diff
@@ -17,6 +17,11 @@ import (
 	"crypto/rsa"
 	"flag"
 
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/cloud/go/services/bootstrapper/servicers/registration"
+	"magma/orc8r/cloud/go/sqorc"
+	storage2 "magma/orc8r/cloud/go/storage"
```
Comment (hcgatewood) [inline orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go:23]: Looks like no need to alias

File: orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go [hunk 2]
```diff
@@ -34,26 +39,60 @@ var (
 func main() {
 	srv, err := service.NewOrchestratorService(orc8r.ModuleName, bootstrapper.ServiceName)
 	if err != nil {
-		glog.Fatalf("Error creating service: %+v", err)
+		glog.Fatalf("error creating service: %+v", err)
+	}
+
+	bs := createBootstrapperServicer()
+	crs, rs := createRegistrationServicers()
```
Comment (hcgatewood) [inline orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go:46]: Var names are too abbreviated -- "bootstrapperServicer" eg

File: orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go [hunk 3]
```diff
@@ -34,26 +39,60 @@ var (
...
-	protos.RegisterBootstrapperServer(srv.GrpcServer, servicer)
+	return servicer
+}
 
-	err = srv.Run()
+func createRegistrationServicers() (protos.CloudRegistrationServer, protos.RegistrationServer) {
+	db, err := sqorc.Open(storage2.GetSQLDriver(), storage2.GetDatabaseSource())
 	if err != nil {
-		glog.Fatalf("Error running service: %+v", err)
+		glog.Fatalf("failed to connect to database: %s", err)
```
Comment (hcgatewood) [inline orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go:78]: Always template err with `%+v` to give full info eg stack trace

Comment (reginawang3495) [reply orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go:79]: there's a ton of `%s` for err everywhere?

Comment (hcgatewood) [reply orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go:79]: Correct, there are 🙂

**Review by reginawang3495 — COMMENTED** [review]
forgot to submit review!

**Pull request closed without merging**